src/dropdown/src/DropdownOption.tsx

```diff
@@ -52,6 +53,10 @@ export default defineComponent({
     placement: {
       type: String as PropType<FollowerPlacement>,
       default: 'right-start'
+    },
+    optionProps: {
+      type: Object as PropType<DropdownOptionProps | null>,
```

Comment:
```suggestion
      type: Object as PropType<DropdownOptionProps | null>,
```

这个地方应该是 HTMLDivAttributes 之类的东西，从 vue 中导出的

Reply:
default 不是 null，直接写 props: Object 就行了，默认是 undefined